torch/csrc/jit/serialization/python_print.cpp

```diff
@@ -743,33 +743,9 @@ struct PythonPrintImpl {
     }
   }
 
-  void checkVersion(Node* node) {
```

Comment:
Did you check how this function is called? I remember it does some complicated traverse of the graph to find each node. I could be wrong tho

Reply:
https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/serialization/python_print.cpp#L1255 like this

Reply:
this bit is reverted